arcaflow_plugin_utilities/utilities_plugin.py

```diff
@@ -3,16 +3,25 @@
 import sys
 import uuid
 from datetime import datetime, timezone
+import time
 import typing
 from dataclasses import dataclass
 from arcaflow_plugin_sdk import plugin, schema
 
 
+# For steps that do not require input
 @dataclass
 class InputParams:
```

Comment:
Docstrings for Python classes are placed (indented) on the line after the `class` keyword ([ref](https://peps.python.org/pep-0257/#what-is-a-docstring)).